src/widgets/icons.js

```diff
@@ -57,3 +66,17 @@ ConfirmIcon.propTypes = { style: PropTypes.object };
 export const LockIcon = React.memo(({ style }) => <FAIcon name="lock" size={28} style={style} />);
 LockIcon.defaultProps = { style: { color: FINALISED_RED, marginHorizontal: 8, bottom: 6 } };
 LockIcon.propTypes = { style: PropTypes.object };
+
+export const ChevronDownIcon = React.memo(({ color, size }) => (
+  <FA5Icon name="chevron-down" color={color} size={size} />
+));
+ChevronDownIcon.defaultProps = { color: WHITE, size: 20 };
+ChevronDownIcon.propTypes = { color: PropTypes.string, size: PropTypes.number };
+
+export const PencilIcon = React.memo(({ color, size }) => (
+  <FAIcon name="pencil" color={color} size={size} />
+));
+PencilIcon.defaultProps = { color: WHITE, size: 20 };
+PencilIcon.propTypes = { color: PropTypes.string, size: PropTypes.number };
+
+export const HistoryIcon = React.memo(() => <FA5Icon name="history" />);
```

Comment:
I don't see the need for the `memo` on this guy

Reply:
Functional components by default basically are implementing `shouldComponentUpdate` where it will always return true. Wrapping it in `memo` implements a `shouldComponentUpdate` that does a shallow comparison on all props. Since it doesn't have props this is the same as returning false every call, so it will never re-render v always re-rendering, without memo! 

Memo basically has higher effectiveness, the less frequent the props change. Since the props never change, it is at maximum efficiency!

Maybe it's stupid and premature since it's a little component and adding the memo is just additional code that makes pretty small difference. I thought SVGs would have slightly higher re-draw implications than your average native component?